dts/bindings/display/nxp,imx-elcdif.yaml

```diff
@@ -102,3 +102,9 @@ properties:
         - "24-bit"
       description:
         LCD data bus width. The default is set to the reset value of 16-bit
+
+    backlight-gpios:
```

Comment:
nit: you can also use the regulator framework so that devices don't need any modification (if never controlled).

Reply:
It is in fact driving the backlight that causes issues- see here: https://github.com/zephyrproject-rtos/zephyr/issues/26527#issuecomment-678524116. We cannot use the regulator framework for this.

Reply:
np, was just a suggestion

Reply:
No worries, just glad I checked- I wasn't aware of the full extend of the backlight issue, and the regulator approach would have been cleaner if the backlight problem wasn't present.